models/workitem_repository.go

```diff
@@ -191,7 +191,7 @@ func (r *GormWorkItemRepository) List(ctx context.Context, criteria criteria.Exp
 	return result, nil
 }
 
-func convertFromModel(wiType WorkItemType, workItem WorkItem) (*app.WorkItem, error) {
+func ConvertFromModel(wiType WorkItemType, workItem WorkItem) (*app.WorkItem, error) {
```

Comment:
I would consider moving this method to the WorkItemType struct as a member, analogous to the Field.ConvertFromModel() method.